app/interface_student.py

```diff
@@ -0,0 +1,165 @@
+from aiogram import Bot, types
+from app.states import get_user_state, set_user_state, get_user_data, set_user_data, clear_user_data
+from app.api.student import Student
+''', StudentJoinGroupException, StudentLeaveGroupException'''
+
+
+class StudentInterface:
+    def __init__(self, bot: Bot, student_name: str):
+        self.bot = bot
+        #self.database = Database() #добавить
+        self.student = Student(student_name)
```

Comment:
Seems like you missed second 'database' parameter in 'Student' constructor

Reply:
fixed